test/services/homework/submissionService.test.js

```diff
@@ -150,4 +151,76 @@ describe('submission service', function test() {
 		expect(result).to.haveOwnProperty('_id');
 		expect(result.grade).to.eq(100);
 	});
+
+	it('teacher can FIND all submissions on the school', async () => {
+		const { _id: schoolId } = await testObjects.createTestSchool();
+		const [originalTeacher, teacher , student] = await Promise.all([
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no space before ','.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5502439)